### PR TITLE
Add 'Möbel-Kraft Online GmbH & Co. KG' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22moebel-kraft%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22categories%22%3A%5B%22commerce%22%5D%2C%22name%22%3A%22M%C3%B6bel-Kraft%20Online%20GmbH%20%26%20Co.%20KG%22%2C%22address%22%3A%22Am%20Rondell%201%5Cn12529%20Sch%C3%B6nefeld%20%22%2C%22phone%22%3A%2249%20(0)4551%20%2F%2050%20-1%22%2C%22email%22%3A%22onlineservice%40moebel-kraft.de%22%2C%22web%22%3A%22https%3A%2F%2Fwww.moebel-kraft.de%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.moebel-kraft.de%2Fimpressum%22%2C%22https%3A%2F%2Fwww.moebel-kraft.de%2Fdatenschutz%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1298%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/moebel-kraft.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "moebel-kraft",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Möbel-Kraft Online GmbH & Co. KG",
+    "address": "Am Rondell 1\n12529 Schönefeld ",
```
Comment (moriarty-jim) [inline companies/moebel-kraft.json:10]:
```suggestion
    "address": "Ziegelstraße 1\n23795 Bad Segeberg\nDeutschland",
```
corrected address, see [https://www.moebel-kraft.de/datenschutz](https://www.moebel-kraft.de/datenschutz)

File: companies/moebel-kraft.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "phone": "49 (0)4551 / 50 -1",
+    "email": "onlineservice@moebel-kraft.de",
```
Comment (moriarty-jim) [inline companies/moebel-kraft.json:12]:
```suggestion
    "email": "datenschutz@moebel-kraft.de",
```
DPO email

File: companies/moebel-kraft.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "slug": "moebel-kraft",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Möbel-Kraft Online GmbH & Co. KG",
+    "address": "Am Rondell 1\n12529 Schönefeld ",
+    "phone": "49 (0)4551 / 50 -1",
```
Comment (moriarty-jim) [inline companies/moebel-kraft.json:11]:
```suggestion
    "phone": "+49 4551 501",
```
fixed phone format

File: companies/moebel-kraft.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "email": "onlineservice@moebel-kraft.de",
+    "web": "https://www.moebel-kraft.de",
```
Comment (moriarty-jim) [inline companies/moebel-kraft.json:13]:
```suggestion
    "fax": "+49 08003522222",
    "web": "https://www.moebel-kraft.de",
```
added generic fax
I'm not quite sure, if this is the correct international format for: +49 (0) 0800 - 35 22 222
source: [https://www.moebel-kraft.de/impressum](https://www.moebel-kraft.de/impressum)

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
changes to address, email; fixed phone; added fax

File: companies/moebel-kraft.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    "sources": [
+        "https://www.moebel-kraft.de/impressum",
+        "https://www.moebel-kraft.de/datenschutz",
+        "https://github.com/datenanfragen/data/pull/1298"
+    ],
```
Comment (baltpeter) [inline companies/moebel-kraft.json:18]:
```suggestion
    ],
    "suggested-transport-medium": "email",
```

File: companies/moebel-kraft.json (hunk 4 above)
Comment (baltpeter) [reply companies/moebel-kraft.json:13]: It should be `+49 800 3522222` (I'll add that above).

We use https://libphonenumber.appspot.com/ ("International format" under "Formatting Results" on the results page) as our canonical source for phone number formatting (but additional spaces can be added in the last block for easier readability).

File: companies/moebel-kraft.json (hunk 3 above)
Comment (baltpeter) [reply companies/moebel-kraft.json:11]:
```suggestion
    "phone": "+49 4551 501",
    "fax": "+49 800 3522222",
```

**Review by baltpeter — APPROVED** [review]
Thanks, everyone.

File: companies/moebel-kraft.json (hunk 4 above)
Comment (moriarty-jim) [reply companies/moebel-kraft.json:14]: I have never seen 00800 as prefix before, only 0800 for toll-free numbers (or any area code/prefix with double 0)...

Comment (baltpeter) [reply companies/moebel-kraft.json:14]: I think that's a typo on the site. Pretty sure that isn't valid otherwise.

Comment (moriarty-jim) [reply companies/moebel-kraft.json:14]: I just found out that 00800 actually exists 👀 , see [00800: Free­phone inter­national (teltarif.de)](https://www.teltarif.de/i/sonderrufnummern-0800.html)

Comment (baltpeter) [reply companies/moebel-kraft.json:14]: Interesting, didn't know that either.

Comment (moriarty-jim) [reply companies/moebel-kraft.json:14]: Anyhow, no idea how you would call 00800 from abroad. +49 0800 or +49 800 ...

**Pull request merged**